packages/web-twig/Makefile

```diff
@@ -80,8 +80,33 @@ encore-install: ## Install demo dependencies
 	cd $(APP_DOCKER_DIR) && $(DOCKER_COMP) exec $(DOCKER_ENCORE_SERVICE) yarn install
 
 encore-link: ## Link demo dependencies
-	cd $(APP_DOCKER_DIR) && $(DOCKER_COMP) exec $(DOCKER_ENCORE_SERVICE) sh -c "cd /srv/spirit-web && yarn link && cd /srv/spirit-web-twig-demo && yarn link @lmc-eu/spirit-web && cd /srv/spirit-form-validations && yarn link && cd /srv/spirit-web-twig-demo && yarn link @lmc-eu/spirit-form-validations && cd /srv/spirit-demo && yarn link && cd /srv/spirit-web-twig-demo && yarn link @lmc-eu/spirit-demo"
-
+	cd $(APP_DOCKER_DIR) && \
```

Comment:
Can we document this somewhere somehow?

Reply:
Can you be more specific on what needs to be documented? Since I am the author I cannot distinguished what is the case that is not understandable :-/

Reply:
Explain what is happening and what might be the problem if it again occurs.